frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -79,39 +81,43 @@ export default function SideNavigation({
   page,
   currentUser,
   toggleDrawer,
-  onError,
   onResetErrorMessage,
   drawerOpen = false,
   pageTopStyling = {},
 }: SideNavigationProps) {
   const [reviewCount, setReviewCount] = useState(0)
   const theme = useTheme()
+  const { reviews, isReviewsLoading, isReviewsError } = useGetReviewRequestsForUser()
+
+  const doesNotContainUserResponse = useCallback(
+    (review: ReviewRequestInterface) => {
+      return (
+        currentUser &&
+        review.responses.filter(
```

Comment:
Slight performance improvement if you use a `find` so it exits early if it can:
```
      return (
        currentUser &&
        !review.responses.find(
          (response) => response.user === `user:${currentUser.dn}` && response.decision === Decision.Approve,
        )
      )
```